test/unit/modules/request-browser-spec.js

```diff
@@ -86,6 +86,151 @@ describe('Browser Request methods', () => {
         });
     });
 
+    const newConfig = {...helpers.client, roundValues: true};
+    const customClient = new KeenClient(newConfig);
+    const dummyNormalResponseUp = { result: 14.612}
+    const dummyRoundResponseUp = { result: 15 }
+
+    it('rounding values up', async () => {
+      fetch.mockResponseOnce(JSON.stringify(dummyNormalResponseUp));
+      const queryParams = {
+        analysis_type: 'average',
+        event_collection: 'pageviews',
+        timeframe: 'this_3_months'
+      };
+      await customClient
+        .query(queryParams)
+        .then(res => {
+          expect(res.result).toBe(dummyRoundResponseUp.result);
+        });
+    });
+
+    const dummyNormalResponseDown = { result: 14.212}
+    const dummyRoundResponseDown = { result: 14 }
```

Comment:
let's move both of these CONSTs to the line :112 (inside the IT function), because these values are used only by this function